deploy/olm-catalog/open-liberty-operator-certified/open-liberty-operator.v0.5.1.clusterserviceversion.yaml

```diff
@@ -1,16 +1,16 @@
 apiVersion: operators.coreos.com/v1alpha1
```

Comment:
Could you please add this into a new file? We should keep 0.5.0 as is for reference. Thanks.